dashboard/db/migrate/20170131165950_add_uniqueness_to_user_proficiency_user_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniquenessToUserProficiencyUserId < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
ah, I'm afraid not, although I would expect an operation that has order-specific intermediate steps like this one to not easily translate to a `change`